speech/tests/unit/test__gax.py

```diff
@@ -16,6 +16,9 @@
 
 import mock
 
+from google.cloud import _helpers
+from google.cloud.gapic.speech.v1.speech_client import SpeechClient
```

Comment:
> If we decide to break from the style, it should be wholesale, not ad-hoc.

This is a hugely circular argument (especially given staffing considerations). It basically makes it nearly impossible to change anything because you have to go back and change it across a huge spectrum of code.

It is especially unimportant in this case since this is code that is _literally going away_ in two months and only being kept around for backwards compatibility so users get some time to move. This file will be deleted wholesale before the end of the quarter.

All that is to say: I am not going to refactor this. :-)